packages/lib-classifier/src/components/Classifier/hooks/usePanoptesUserSession.js

```diff
@@ -6,7 +6,7 @@ import {
 
 export default function usePanoptesUserSession({ authClient, projectID }) {
   const { data: user, isLoading: userLoading } = usePanoptesUser(authClient)
-  const userID = !userLoading && user?.id
+  const userID = userLoading ? undefined : user?.id
```

Comment:
I'm not sure I'm thinking about this correctly, but I think 
- if the user is loading we want `userID` to be undefined ✔️ , 
- if the user isn't loading then there are two possible scenarios
  - we have a user in which case we want `userID` to be the user.id ✔️  
  - we want `userID` to be null ✖️ (for an anonymous / logged out user) which I think the following change would do...

```suggestion
  const userID = userLoading ? undefined : user?.id || null
```

Reply:
Ha! I just pushed exactly this change, then saw your comment.

Reply:
But yeah, you’re exactly right. We need to be able to distinguish between a value that’s falsy because it’s still loading and a value that’s falsy because you aren’t actually logged in.

Reply:
This change gets the tests passing again for anonymous users. Without it, you never get past ‘Loading…’.